fix `Contributing` wording (#1426)
microsoft/vscode-loc: README.md

## Contributing

> If you want to give feedback or report issue with a translation, please create a [new GitHub issue](https://github.com/microsoft/vscode-loc/issues/new). Please check if a topic about your issue already exists! Translation strings are managed and edited in Microsoft Localization Platform and changes to strings can only be made there. Due to this, pull request fixing translations won't be accepted. All other PRs to things like `README.md`, `package.json`, etc will be accepted at the discretion of the maintainers.

Reviewer: Even moar fixes
```suggestion
If you want to give feedback or report an issue with a translation, please create a [new GitHub issue](https://github.com/microsoft/vscode-loc/issues/new). Please check if a topic about your issue already exists! Translation strings are managed on the Microsoft Localization Platform, and changes to strings can only be made there. Consequently, pull requests fixing translations won't be accepted. All other PRs to things like `README.md`, `package.json`, etc. will be accepted at the discretion of the maintainers.
```

> If you want to give feedback or report an issue with a translation, please create a [new GitHub issue](https://github.com/microsoft/vscode-loc/issues/new). Please check if a topic about your issue already exists! Translation strings are managed on the Microsoft Localization Platform, and changes to strings can only be made there. Consequently, pull requests fixing translations won't be accepted. All other PRs to things like `README.md`, `package.json`, etc. will be accepted at the discretion of the maintainers.

Author: ```suggestion
If you want to give feedback or report an issue with a translation, please create a [new GitHub issue](https://github.com/microsoft/vscode-loc/issues/new). Please check if a topic about your issue already exists!
Translation strings are managed on the Microsoft Localization Platform, and changes to strings can only be made there. Consequently, pull requests fixing translations won't be accepted. All other PRs to things like `README.md`, `package.json`, etc. will be accepted at the discretion of the maintainers.
```
